src/test/java/com/cdancy/bitbucket/rest/features/BranchApiLiveTest.java

```diff
@@ -136,6 +140,48 @@ public void testDeleteBranchPermission() {
         }
     }
 
+    @Test(dependsOnMethods = {"testCreateBranch", "testListBranches"})
+    public void testGetBranchModelConfiguration() {
+        branchModelConfiguration = api().getModelConfiguration(projectKey, repoKey);
+        checkDefaultBranchConfiguration();
+    }
+
+    @Test(dependsOnMethods = {"testCreateBranch", "testListBranches"})
+    public void testGetBranchModelConfigurationOnError() {
+        BranchModelConfiguration configuration = api().getModelConfiguration(projectKey, "12345");
+        assertThat(configuration.errors()).isNotEmpty();
+        assertThat(configuration.development()).isNull();
+        assertThat(configuration.production()).isNull();
+        assertThat(configuration.types()).isEmpty();
+    }
+
+    private void checkDefaultBranchConfiguration() {
+        assertThat(branchModelConfiguration).isNotNull();
+        assertThat(branchModelConfiguration.errors().isEmpty()).isTrue();
+        assertThat(branchModelConfiguration.development().refId()).isNotNull();
```

Comment:
This was actually null when I ran the tests. Seeing as how annotated with @Nullable not sure if we should check for it here? I'm going to merge this now and remove the check. If you have a strong objection to this let me know and we can follow up on #69